tests/kuttl/tests/dataplane-create-test/00-dataplane-create.yaml

```diff
@@ -41,7 +41,7 @@ spec:
          service_net_map:
            nova_api_network: internal_api
            nova_libvirt_network: internal_api
-         edpm_chrony_ntp_servers:
+         edpm_ntp_servers:
```

Comment:
My reading of https://github.com/openstack-k8s-operators/edpm-ansible/pull/373 makes
me think the correct way to override the default time server is this:
```
edpm_ntp_servers:
  - hostname: 'clock.redhat.com'
    max_poll: 10
    min_poll: 6
```
Will this list of items instead of a map work?

`clock.redhat.com vs {hostname: clock.redhat.com}`

Reply:
Not really, the poll aregs have defaults set to 10/6 in timesync, just like edpm_chrony. We don't really need to set them unless we decide those don't work for some reason.

As for the `hostname`, this will be handled on the edpm-ansible side.

Reply:
I think you're saying that `edpm_ntp_servers:` ['clock.redhat.com']` will also work.

Reply:
I'm not going to count on it. But I'll make the necessary changes in edpm-ansible, rather than here. That way we can have something that is compatible with both edpm_chrony and timesync, minimizing chances of breakage, later we can move toward more timesync-like form.

Reply:
I wouldn't expect `edpm_ntp_servers: ['clock.redhat.com']` to work so that's why I asked. IMO better to just force the new format rather than remain backwards compatible since none of this is marked stable yet. Else, we'll have to maintain the translation to support the old format but it won't be worth it IMO.

Reply:
I don't understand the need for edpm_ntp_servers at all. Unless it is just an intermediate step? We shouldn't have EDPM variables wrapping variables from the system roles unless absolutely necessary for some reason. Just use the timesync_ntp_server variable directly.

Reply:
That can be done. But given that there may be other cases of the role being used I  thought it safer to denote clearly what this one var is supposed to be doing. If we just switch to the same name role uses, it won't be apparent at first glance.

Reply:
Variable renamed.